tests/python_tests/test_eltwise_unary_datacopy.py

```diff
@@ -10,37 +11,42 @@ def generate_golden(operand1, format):
     return operand1
 
 
-param_combinations = [
-    (format, dest_acc, testname)
-    for format in ["Float32", "Bfp8_b", "Float16_b", "Float16", "Int32"]
-    for dest_acc in ["", "DEST_ACC"]
-    for testname in ["eltwise_unary_datacopy_test"]
-]
-
-param_ids = [f" format={comb[0]} | dest_acc={comb[1]} " for comb in param_combinations]
+full_sweep = False
+all_format_combos = generate_format_combinations(
+    ["Float32", "Bfp8_b", "Float16_b", "Float16", "Int32"], not full_sweep
```

Comment:
```suggestion
    formats = ["Float32", "Bfp8_b", "Float16_b", "Float16", "Int32"], all_same = True
```
This looks easier to understand to the untrained eye.

Reply:
Agreed! Also follows function parameter construct better